client/source/js/modules/d3-charts/stacked-area-directive.js

```diff
@@ -159,6 +159,9 @@ define(['./module', './scale-helpers', 'angular', 'underscore'], function (modul
       );
 
       d3Charts.drawTitleAndLegend(svg, options, headerGroup);
+
+      var svgHeight = parseInt($(svg[0][0]).css('height'), 10);
+      rootElement.trigger('draw', [{height: svgHeight}]);
```

Comment:
With this you are coupling the `stacked-area-chart` with the `normalise-heigh` directive.

Is loosely coupled so is not that bad but, before killing it, you had the `after-render` directive that you could have use to trigger whatever event you want from the view using the chart like:  `<stackedAreaChart  ...  after-render="$emit('element-rendered')">